### PR TITLE
Make node clean retain output directories.

### PR DESCRIPTION
**Pull request by jkinkead**
yay.

### CODE REVIEW

File: gulp/index.js [hunk 1]
```diff
@@ -305,9 +305,9 @@ module.exports = {
      * Removes all build artifacts.
      */
     gulp.task('clean', function(cb) {
-      gutil.log(util.format('Cleaning: %s', gutil.colors.magenta(paths.build)));
-      var targets = [ paths.build ];
-      return del(targets, { force: true });
+      var target = paths.build + '/**/*';
```
Comment (codeviking) [inline gulp/index.js]: What if the user specifies a glob pattern, ie:

``
../foo/bar/**/*
```

It seems like we might want to just change the default path and let users ultimately indicate if they want to delete the entire directory?

Comment (codeviking) [inline gulp/index.js]: Wait, users won't do that, as they don't want to set `build` to a glob pattern :).   This seems fine.  We might want to eventually expose a `paths.clean` key, but we don't need to do that now.

**Comment by codeviking** [comment]
I corrected myself -- this is good to go.

File: gulp/index.js (hunk 1 above)
Comment (jkinkead) [inline gulp/index.js]: Cool, that SGTM!

**Pull request merged**